keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -287,9 +298,8 @@ CompletedIndexes: "2-4,6-7"
 The `kubectl describe` command crops the list of indexes if it's too long:
 
 ```
-Completed Indexes: [1-25,28,30-32,...]
+Completed Indexes: 1-25,28,30-32,...
```

Comment:
I'm assuming, that to propagate this field, we will be using the exact same mechanism that is proposed in: https://github.com/kubernetes/enhancements/pull/2308 (i.e. the untracked... mid step, right?)

If so, I think it's worth mentioning that explicitly and pointing out this dependency very explicitly.

Reply:
Yes, but that's going to be for beta. For alpha, I'll only track with legacy behavior (lingering Pods). This is point out in the other KEP.

Reply:
So is this field doing to be propagated in alpha at all?

Reply:
I assume you mean populated.

Yes, it will, from the Pods that were not deleted.